backend/src/main.py

```diff
@@ -48,6 +63,9 @@ def setpassword():
 
 @app.route("/login", methods=["POST"])
 def login():
+    # email sending
+    entities.mail.send_mail()
```

Comment:
This is a sample function, convert this into actual functionality to send a actual meaning ful message to the user, 
the input to the email function should be a list of To list, a list of CC list and Subject in string and Body as a object or a rich text to send email with images etc in the future. 

for example, when the time is submitted by the employee to some manager, then the manager should get a message that the time has been submitted for xx hours by yy employee. 
also when the manager approves the time, the employee should get a mail saying that your time for xx days has been apprvoed by yy person.

implement this first for time submission.

Reply:
Updated code. please check.